sdk/requirements.txt

```diff
@@ -2,4 +2,6 @@ urllib3 >= 1.26
 six >= 1.10
 certifi >= 2021.10.8
 python-dateutil >= 2.5.3
-requests >= 2.26.0
\ No newline at end of file
+requests >= 2.26.0
+parameterized~=0.8.1
```

Comment:
looks like this isnt needed as its in the requirements.dev.txt (failed the build)